htmresearch/frameworks/nlp/classification_model.py

```diff
@@ -416,3 +420,19 @@ def _inferDocumentDetailed(self, tokenList, sortResults=True):
 
     else:
       return categoryVotes, sampleIdList, distancetoSampleIds
+
+
+  @staticmethod
+  def _sortArray(array, seed=42):
```

Comment:
Yes, we want ties between items to be broken without always giving precedence to earlier items, but still get repeatable results.